src/services/data/comments.ts

```diff
@@ -62,6 +62,51 @@ export default (client, httpAuth: HttpInstance) => ({
     ).data;
   },
 
+  /**
+   * Find By Id
+   * @param id the Id to search for
+   * @param schemaId the schema Id
+   * @param documentId the document Id
+   * @param rql an optional rql string
+   * @returns the first element found
+   */
+  async findById(
+    id: ObjectId,
+    schemaId: ObjectId,
+    documentId: ObjectId,
+    rql?: RQLString
+  ): Promise<Comment> {
+    const rqlWithId = rqlBuilder(rql).eq('id', id).build();
+    const res = (
+      await client.get(
```

Comment:
Any preference to do it like vs for example something like:
```ts
  async findById(id: ObjectId, rql?: RQLString): Promise<Schema> {
    const rqlWithId = rqlBuilder(rql).eq('id', id).build();
    const res = await this.find({ rql: rqlWithId});
    return res.data[0];
  },
```

Reply:
that's what I wanted to do, but unfortunately `this` keyword refers to the calle.

Reply:
all done! thanks!